src/Tmds.MDns/NetworkInterfaceHandler.cs

```diff
@@ -488,6 +493,11 @@ private void HandlePacketServiceInfos()
                     {
                         service = packetService;
                         _serviceInfos.Add(packetName, service);
+                        if (!_serviceHandlers.ContainsKey(packetName.SubName(1)))
+                        {
+                            continue;
```

Comment:
Similar, the _packetServiceInfos should only contain services for which we have a handler.
This service should not have made it into the _packetServiceInfos.